internal/run/run.go

```diff
@@ -35,7 +35,7 @@ func Start(tunnel tunnel.Tunneler, lbClient *lb.Client, nodeClient node.Client,
 	}
 	log.Infof("Registered to load balancer %s", lbClient.BaseURL.String())
 
-	go tunnel.StartTunnel(id, registerResponse.TunnelURL, registerResponse.Token)
+	go tunnel.StartTunnel(id, registerResponse.TunnelURL, registerResponse.Token, registerResponse.Port)
```

Comment:
change param order lel

Reply:
Well Port is not related to TunnelURL so param order is because of that